docs/content/guides/styling/themes/javascript/exampleTheme.js

```diff
@@ -1288,3 +1288,25 @@ new Handsontable(example, {
   navigableHeaders: true,
   licenseKey: 'non-commercial-and-evaluation',
 });
+
+const colorBox = document.getElementById('colorBox');
+const themeSelect = document.getElementById('themeSelect');
+const currentTheme = document
+  .querySelector('html')
+  ?.classList.contains('theme-dark')
+  ? 'ht-theme-horizon-dark'
+  : 'ht-theme-horizon';
+
+const setTheme = (theme) => {
+  colorBox.classList.value = `color-box ${theme}`;
+  hotInstance?.useTheme(theme);
+  hotInstance.render();
```

Comment:
The `useTheme` method always exists in the Handsontable instance
```suggestion
  hotInstance.useTheme(theme);
  hotInstance.render();
```